frontend/eda/access/common/TeamAccess.tsx

```diff
@@ -1,176 +1,26 @@
-          teamAccess.summary_fields.role_definition.description,
-        card: 'description',
-        list: 'description',
-      },
-    ],
-    [t]
-  );
-  const toolbarFilters = useMemo<IToolbarFilter[]>(
-    () => [
-      {
-        key: 'team__name',
-        label: t('Team name'),
-        type: ToolbarFilterType.SingleText,
-        query: 'team__name',
-        comparison: 'equals',
-      },
-      {
-        key: 'role_definition__name',
-        label: t('Role name'),
-        type: ToolbarFilterType.SingleText,
-        query: 'role_definition__name',
-        comparison: 'equals',
-      },
-    ],
-    [t]
-  );
-  const view = useEdaView<TeamAssignment>({
-    url: edaAPI`/role_team_assignments/`,
-    tableColumns,
-    toolbarFilters,
-    queryParams: { object_id: id, content_type__model: type },
-  });
-  const removeRoles = useRemoveRoles(view.unselectItemsAndRefresh);
-
-  const rowActions = useMemo<IPageAction<TeamAssignment>[]>(
-    () => [
-      {
-        type: PageActionType.Button,
-        selection: PageActionSelection.Single,
-        variant: ButtonVariant.primary,
-        icon: MinusCircleIcon,
-        isPinned: true,
-        label: t('Remove role'),
-        onClick: (item: TeamAssignment) => removeRoles([item]),
-      },
-    ],
-    [t, removeRoles]
-  );
-  const toolbarActions = useMemo<IPageAction<TeamAssignment>[]>(
-    () => [
-      {
-        type: PageActionType.Link,
-        selection: PageActionSelection.None,
-        variant: ButtonVariant.primary,
-        isPinned: true,
-        icon: PlusCircleIcon,
-        label: t('Add roles'),
-        href: getPageUrl(addRolesRoute ?? '', { params: { id: id } }),
-      },
-      {
-        type: PageActionType.Button,
-        selection: PageActionSelection.Multiple,
-        icon: TrashIcon,
-        label: t('Delete selected roles'),
-        onClick: (items: TeamAssignment[]) => removeRoles(items),
-        isDanger: true,
-      },
-    ],
-    [t, getPageUrl, addRolesRoute, id, removeRoles]
-  );
+  const { t } = useTranslation();
   return (
-    <PageTable
-      id="eda-team-access-table"
-      tableColumns={tableColumns}
-      toolbarActions={toolbarActions}
-      toolbarFilters={toolbarFilters}
-      rowActions={rowActions}
-      errorStateTitle={t('Error loading role access data.')}
-      emptyStateTitle={t('There are currently no roles assigned to this object.')}
-      emptyStateDescription={t('Please add a role by using the button below.')}
-      emptyStateButtonText={t('Add role')}
-      emptyStateActions={toolbarActions.slice(0, 1)}
-      {...view}
-      defaultSubtitle={t('Team access')}
+    <Access
+      tableColumnFunctions={{
+        name: {
+          function: (teamAccess: TeamAssignment) => teamAccess.summary_fields.team.name,
+          sort: 'team__name',
+          label: t('Team name'),
+        },
+      }}
+      toolbarFiltersValues={{ label: '', query: '' }}
```

Comment:
The `Team Access` filters do not appear to be working. We probably need to provide initial values here? 
`toolbarFiltersValues={{ label: t('Team name'), query: 'team__name' }}`

Reply:
Fixed. Thanks!